app/views/devise/sessions/new.html.erb

```diff
@@ -1,24 +1,61 @@
- 
-<div class="container">
-  <h2>Log In</h2>
-  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-    <div class="field">
-      <%= f.label :email %><br />
-      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-    </div>
-    <div class="field">
-      <%= f.label :password %><br />
-      <%= f.password_field :password, autocomplete: "current-password" %>
-    </div>
-    <% if devise_mapping.rememberable? %>
-      <div class="field">
-        <%= f.check_box :remember_me %>
-        <%= f.label :remember_me %>
+<div class="community_post pb-5">
+    <div class="row d-flex justify-content-center align-items-center">
+      <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+        <div class="card text-dark" style="border-radius: 1rem;">
+          <div class="card-body p-5 text-center">
+            <div class="text-left">
+              <h5 class="fw-bold mb-2">Log In</h5>
+              <p class="text-dark-50 login-head-text mb-4">By continuing, you agree are setting up a Reddit account and agree to our User Agreement and Privacy Policy.</p>
+
+
+              <div class="d-flex justify-content-center text-center mt-4 pt-1">
+                <a href="#!" class="text-dark"><i class="fab fa-apple fa-lg"></i></a>
+                <a href="#!" class="text-dark"><i class="fab fa-facebook-f fa-lg mx-4 px-2"></i></a>
+                <a href="#!" class="text-dark"><i class="fab fa-google fa-lg"></i></a>
+              </div>
+
+              <p class="or-line mt-4">OR</p>
+
+              <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+
+                <div class="form-group first">
+                  <%= f.label :email, class: "form-label" %>
+                  <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email" %>
+                </div>
+
+              <div class="form-group last">
+                <%= f.label :password, class: "form-label" %><br />
+                <%= f.password_field :password, class: "form-control", autocomplete: "current-password" %>
+              </div>
+
+              <p class="small">
+              Forgot your
+              <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+                <%= link_to "password", new_account_password_path(resource_name) %>
+              <% end %>
+              ?</p>
```

Comment:
this question mark looks odd. Can you add that with link_to itself? @iMagesh is this idea good?

Reply:
yes @Venirails. @vasanthakumar-a can you add the question mark in the link_to itself?